src/app/modules/main/wallet_section/transactions/controller.nim

```diff
@@ -61,6 +61,9 @@ method init*(self: Controller) =
     let args = TransactionsLoadedArgs(e)
     self.delegate.setTrxHistoryResult(args.transactions, args.address, args.wasFetchMore)
 
+method checkPendingTransactions*(self: Controller) =
```

Comment:
Do we need to expose checkPendingTransaction in controller/module/view. I think it could be something we trigger only from inside the service.

Also I don t see any usage of those function